ui/app/com/gu/recipeasy/controllers/Application.scala

```diff
@@ -30,8 +30,37 @@ class Application(override val wsClient: WSClient, override val conf: Configurat
   }
 
   private def isShowCreation(): Boolean = {
-    // TODO: write the logic to compute the correct answer here
-    true
+
+    // This function indicates whether the user should be redirected to a curation/parsing step or a verification step
```

Comment:
This huge comment is unnecessary. Your code should be (and is to me) self documenting.